test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -39,15 +39,15 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-attacher
-          image: gcr.io/gke-release/csi-attacher:v2.2.0-gke.0
+          image: k8s.gcr.io/sig-storage/csi-attacher:v2.2.0
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
         - name: csi-resizer
-          image: gcr.io/gke-release/csi-resizer:v0.5.0-gke.0
+          image: k8s.gcr.io/sig-storage/csi-resizer:v0.5.0
```

Comment:
I saw there is updated images `v0.5.1` and even `v1.1.0` https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml#L28-L35

should we update to use the path release or upgrade to 1.1.0?

Reply:
Yeah upgrading to 1.1.0 is fine, thanks!

Reply:
/hold
To address this

Reply:
done @spiffxp PTAL

how we will deal with this image `gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver:v1.0.1-gke.0` per my comment https://github.com/kubernetes/kubernetes/pull/98805#pullrequestreview-584302108

Reply:
upgrading to 1.1.0 looks like now we have some timeouts :/